src/mongo/db/server_options_helpers.cpp

```diff
@@ -254,12 +254,12 @@ Status addGeneralServerOptions(moe::OptionSection* options) {
                                "Desired format for timestamps in log messages. One of ctime, "
                                "iso8601-utc or iso8601-local");
 
-#if MONGO_ENTERPRISE_VERSION
+//#if MONGO_ENTERPRISE_VERSION
```

Comment:
Let's not keep commented out code, as I don't think anybody cares that this parameter is in Mongo Enterprise and we don't use their macro definitions.